Assets/Mirror/Runtime/LocalConnections.cs

```diff
@@ -42,7 +42,7 @@ public override void Disconnect()
 
     internal class LocalConnectionBuffer
     {
-        readonly NetworkWriter writer = new NetworkWriter();
+        readonly NetworkWriter writer = new NetworkWriter(ushort.MaxValue);
```

Comment:
would be better to make this an argument of `new LocalConnectionBuffer()` rather an a constant value so it can be changed if needed by users